test/sequential/test-native-module-deps.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+// This tests that all the native modules can be loaded independently
+// Flags: --expose-internals
+
+if (process.argv[3]) {
+  require(process.argv[3]);
+  return;
+}
+
+require('../common');
+const {
+  cachableBuiltins
+} = require('internal/bootstrap/cache');
+const { fork } = require('child_process');
+const assert = require('assert');
+
+for (const key of cachableBuiltins) {
+  run(key);
+}
+
+function run(key) {
+  const child = fork(__filename,
```

Comment:
@refack `-e` introduces noise in the dependency graph because that option also leads to additional module loads. Same goes to `-p`.

Reply:
Then maybe use a fixture?
It seems to me like in this case the self referenced part makes the file look awkward. And the usual benefit of having all the test code in one place, is not that beneficial since the child code is just one expression.
But it's just a style nit, and I defer to your decision.

Reply:
New idea, a variation on (either with shell or by using `child.stdin`):
```js
const child = exec(`echo "require('${key}')" | `${process.execPath}` --`, {shell: true});
```